docs/code/reference.md

```diff
@@ -343,6 +343,95 @@ public override bool configure_event (Gdk.EventConfigure event) {
 }
 ```
 
+# Logging {#logging}
+
+There are various logging methods to use and here are some sample references for you.
+
+## Critical {#critical}
+
+Critical log level is used when there is a severe application failure that causes premature termination and should be investigated immediately.
```

Comment:
I think `that causes premature termination and should be investigated immediately.` belongs more to the `error` than to `critical`. Critical is just a print in the console while error actually exits.

Reply:
Updated!